packages/node-bridge/helpers.ts

```diff
@@ -295,13 +299,15 @@ export function createServerWithHelpers(
       res.send = body => send(req, res, body);
       res.json = jsonBody => json(req, res, jsonBody);
 
-      await handler(req, res);
-    } catch (err) {
-      if (err instanceof ApiError) {
-        sendError(res, err.statusCode, err.message);
```

Comment:
I removed the condition for `ApiError` because (1) the split try-catch handles this now and (2) when using `sendError` like this the user experience was bad: the browser showed the raw error page (not our template) and the log showed nothing.

Now, in this case, the log will show:

```
GET /api/node-internal :: Error while handling /api/node-internal: Internal Server Error
```